builder/table.go

```diff
@@ -228,3 +234,15 @@ func (t Table) WriteOptions(buffer *Buffer, options string) {
 	buffer.WriteByte(' ')
 	buffer.WriteString(options)
 }
+
+func (t Table) filterDefinition(table rel.Table) []rel.TableDefinition {
+	result := []rel.TableDefinition{}
+
+	for _, def := range table.Definitions {
+		if t.DefinitionFilter == nil || t.DefinitionFilter(table, def) {
```

Comment:
better to move ` t.DefinitionFilter == nil` as early return before calling this loop

```
if t.DefinitionFilter == nil {
  return table.Definitions
}
```